### PR TITLE
Add: Heatit zig dim250

### PR DESCRIPTION
**Pull request by geirra**
This adds support for the Heatit Zigbee dimmer 250W.

Seems like a oem of  Dimmer-Switch-ZB3.0, but the firmware has some differences.
PowerOnBehavior doesn´t seem to be supported, so have disabled this. 

Tried to keep the config as tiny as possible.

----
```Debug 2023-07-26 09:31:29Received MQTT message on 'zigbee2mqtt/0x3YYYYYYYYYYYY/set' with data '{"power_on_behavior":"previous"}'
Debug 2023-07-26 09:31:29Publishing 'set' 'power_on_behavior' to '0x3YYYYYYYYYYYY'
Error 2023-07-26 09:31:29Publish 'set' 'power_on_behavior' to '0x3YYYYYYYYYYYY' failed: 'Error: Write 0x3YYYYYYYYYYYY/1 genOnOff({"startUpOnOff":255}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNSUPPORTED_ATTRIBUTE')'
Debug 2023-07-26 09:31:29Error: Write 0x3YYYYYYYYYYYY/1 genOnOff({"startUpOnOff":255}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNSUPPORTED_ATTRIBUTE') at Endpoint.checkStatus (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:419:28) at Endpoint.write (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:488:22) at Object.convertSet (/app/node_modules/zigbee-herdsman-converters/src/converters/toZigbee.js:116:13) at Publish.onMQTTMessage (/app/lib/extension/publish.ts:248:36)

### CODE REVIEW

File: src/devices/heatit.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {Definition} from '../lib/types';
+import extend from '../lib/extend';
+
+const definitions: Definition[] = [
+    {
+        fingerprint: [{type: 'Router', manufacturerName: 'Heatit Controls AB', modelID: 'Dimmer-Switch-ZB3.0'}],
+        model: 'Zig Dim 250W',
```
Comment (Koenkk) [inline src/devices/heatit.ts:7]: Is there any better model number written on the device/product site?

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2148

File: src/devices/heatit.ts (hunk 1 above)
Comment (geirra) [reply src/devices/heatit.ts:7]: Hi, this is actually the official name

https://www.thermo-floor.no/heatit-controls/1444420/heatit-zig-dim-250w-zigbee-rotary-dimmer-white-ral-9003

pdf manual

https://documents.heatit.com/1444420/datasheet/eng?_ga=2.217320981.262970775.1690485878-1179559807.1690485878

Open for other suggestions

**Comment by geirra** [comment]
Selected to use the el-number, so this should be ok now @Koenkk

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**